transfer/src/main/java/io/github/fabricators_of_create/porting_lib/transfer/item/ItemStackHandler.java

```diff
@@ -20,7 +21,7 @@
 import net.minecraft.nbt.Tag;
 import net.minecraft.world.item.ItemStack;
 
-public class ItemStackHandler extends SnapshotParticipant<SnapshotData> implements Storage<ItemVariant>, INBTSerializable<CompoundTag> {
+public class ItemStackHandler extends SnapshotParticipant<SnapshotData> implements Storage<ItemVariant>, INBTSerializableCompound {
```

Comment:
This should have been left the same I don't know why INBTSerializableCompound exists

Reply:
alright, I'll revert this here and in create

Reply:
should INBTSerializableCompound be `@Deprecated` then?

Reply:
Yes, I don't even know where it came from

Reply:
mark it for removal actually

Reply:
INBTSerializableCompound exists because generics and injected interfaces don't cooperate. INBTSerializable should only be used when a CompoundTag is not used. Keep both